docs/development_guide.md

```diff
@@ -51,7 +55,56 @@ Note that after running the above command we get some dependency conflicts in
 installed packages which is currently safe to ignore. For details see
 [Issue #71](https://github.com/googlegenomics/gcp-variant-transforms/issues/71).
 
+### Setup IDE
+
+You may choose any IDE as you like. The following steps are intended for
+IntelliJ users.
+
+#### Install IntelliJ IDE
+
+Download
+[IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/download/#section=linux)
+and install.
+
+#### Install Python plugin
+
+Choose File | Settings on the main menu, and then go to Plugins.
+Click the Install JetBrains plugin button.
+In the dialog that opens, search for Python Community Edition and then install
+the plugin.
+
+For more details, refer to
+[Install plugins](https://www.jetbrains.com/help/idea/installing-updating-and-uninstalling-repository-plugins.html).
+
+#### Setup IntelliJ SDK
+
+1. Choose File | Project Structure on the main menu, and then go to Project.
+2. Create a new project SDK by clicking the New button, choose Python SDK then
+add Local.
+3. In the dialog that opens, click the Virtual Environment node. Select New
+environment, and specify the location of the new virtual environment. Note that
+the folder where the new virtual environment should be located must be empty!
+For the Base interpreter, add the python path
+gcp-variant-transforms/venv/bin/python under the created virtualenv.
```

Comment:
I put my venv under gcp-variant-transforms but I think some people don't (@arostamianfar do you do this?). So maybe replace "gcp-variant-transforms" with [PATH_TO_VENV] and mention that this 'venv' directory is what we created in "Setup virtualenv" above.

Reply:
Done.